testsuite/cluster-test/src/instance.rs

```diff
@@ -7,12 +7,38 @@ use anyhow::{ensure, format_err, Result};
 use serde_json::Value;
 use std::{collections::HashSet, ffi::OsStr, fmt, process::Stdio};
 
+#[derive(Clone)]
+pub enum InstanceConfig {
+    Validator(ValidatorConfig),
+    Fullnode(FullnodeConfig),
+}
+
+#[derive(Clone)]
+pub struct ValidatorConfig {
+    pub index: u32,
+    pub num_validators: u32,
+    pub num_fullnodes: u32,
+    pub image_tag: String,
+    pub config_overrides: Vec<String>,
```

Comment:
Can you point to code that reads `config_overrides`? Is this TBD or did I miss this code?

Reply:
I see there is `CFG_OVERRIDES` in shell script, but I don't see where is bridge between those two

Reply:
CFG_OVERRIDES is set as an env variable for validator and it is read by https://github.com/libra/libra/blob/master/docker/validator-dynamic/docker-run-dynamic.sh#L62

Reply:
The env variable is set in the pod spec https://github.com/libra/libra/blob/master/testsuite/cluster-test/src/cluster_swarm/validator_spec_template.yaml#L116

Reply:
But where do you set cfg_overrides to pod equal to `config_overrides` from here?

Reply:
I'm not sure if I completely follow your question.

So `fn upsert_node()` takes an `InstanceConfig` as a parameter which contains `config_overrides`. 

We use that in the pod spec for the node. Its in `cluster_swarm_kube.rs:387`

Reply:
Ok, I see now. Thanks for explanation